.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,11 +1,12 @@
-<!--- This is for filing bugs. If you have a general question, please -->
-<!--- visit https://github.com/nasa/openmct/discussions -->
-
 ---
-name: Bug Report
+name: Bug report
 about: File a Bug !
+title: ''
+labels: type:bug
+assignees: ''
```

Comment:
I don't know about having assignees in the text We use GitHub assignments, and we don't want open source users assigning team members directly anyway?

Reply:
@akhenry this is is leaving the assignees and title as empty. I believe this is required for the template to be detected